tests/playbooks/tests_bond_deprecated.yml

```diff
@@ -97,3 +97,8 @@
             - import_tasks: tasks/remove_test_interfaces_with_dhcp.yml
           tags:
             - "tests::cleanup"
+
+    - name: Restart NetworkManager for the tests to be followed if any
+      service:
+        name: NetworkManager
+        state: restarted
```

Comment:
In what sense is the network down? This probably needs to be fixed in a different way than restarting NM.

Reply:
Sorry, there seems to be some test issue. I cannot reproduce the network issue. I'm removing this reboot code. Thanks.

Reply:
Thanks, @liangwen12year. As put in the closing comment, this pr is no longer needed. It'd be helpful if we have a chance to run all the tests with one ansible-playbook, but so far, there's no such plan...

Reply:
Thanks for notifying.